building/tracks/syllabus/README.md

```diff
@@ -0,0 +1,126 @@
+# Syllabus
+
+A track's [Concept Exercises](/docs/building/tracks/concept-exercises) are exercises designed to teach specific (programming) concepts.
+These concepts form a _syllabus_.
+This documentation contains pointers and tips on how to succesfully design a syllabus for your track.
+
+## The goal of a syllabus
+
+We want to allow students to start writing code immediately, without having to understand everything up front.
+In order to achieve this we hand-wave over details and we leave a lot of things unexplained.
+We simplify where possible and we provide code stubs.
+This reduces the cognitive burden of getting started and provides the time and space for the knowledge to sink in.
+By taking this approach we're not saying that the student doesn't need to know these things, we're saying that they don't need to know them _yet_.
+
+## Basic structure
+
+Exercises are structured as a graph.
+Later exercises teach concepts that depend on having understood concepts that are taught earlier.
+
+## Porting and borrowing
+
+It can be worth looking at how other language tracks have built out their concept exercises.
+That said, if you decide to use other exercises as a starting point for your own, be careful to ensure that the resulting exercise is about the concept as it exists in your language.
+Sometimes concepts differ subtly, sometimes radically.
+Sometimes concepts don't exist at all.
+
+The syllabus, and therefore the concept tree, should represent the concepts that exist in the language.
+
+Don't include concepts just because other tracks do.
+
+In some cases it might be tempting to put a concept in because people often have to work around it by using concepts that do exist.
+Rather than doing this, introduce the concept that the language _does_ use, and consider adding an exercise that explains how to use it in that type of situation.
+
+For example, in Go there are no enums.
+Instead the Go concept tree introduces constants, and teaches how to use constants in the type of situation where you might use enums in other languages.
+
+## Asking for help
+Don't hesitate to ask for help.
```

Comment:
That's an excellent question, actually.

Do we have a channel in the Slack that is dedicated to general syllabus/learning mode stuff?

Could we make a GitHub team that has some individuals in it who are happy to help answer questions about concept exercises? (I'd be happy to be on that team, though I'm still learning about all the nuances in this).

@ErikSchierboom Thoughts?

Reply:
Yeah, @iHiD or I need to create the teams.

I think maybe it could be worth making the channel _and_ creating the team. Suspenders _and_ a belt, so to speak...

Reply:
I've made a team called @exercism/learning-mode-consultants and a Slack channel called #learning-mode. We can change these, of course!

I'm the only member of the team at the moment, but if you have experience developing learning mode for a track and are happy to get pinged with questions about developing a syllabus or concept exercises let me know and you can be added here as well.